apps/playground/src/app/components/ag-grid/editable-grid/editable-grid.module.ts

```diff
@@ -25,5 +23,5 @@ ModuleRegistry.registerModules([AllCommunityModule]);
   ],
 })
 export class EditableGridModule {
-  public static routes = EditInModalGridRoutingModule.routes;
+  public static routes = EditableGridRoutingModule.routes;
```

Comment:
This was another playground issue I unearthed while trying to make sure my stuff was error-free.